metals/src/main/scala/scala/meta/internal/metals/SemanticTokenCapability.scala

```diff
@@ -0,0 +1,59 @@
+package scala.meta.internal.metals
+
+import scala.collection.JavaConverters._
+
+import org.eclipse.lsp4j._
+
+object SemanticTokenCapability {
+  var TokenTypes: List[String] = List(
```

Comment:
1. `var` -> `val`
2. Also, when trying to find index of token type, we can use `val typeIndex = Map[String,Int] = TokenTypes.zipWithIndex.toMap` instead of List's `indexOf` method. 

Same thing (1,2) can be done with token modifiers